packit_service/worker/helpers/sync_release/propose_downstream.py

```diff
@@ -68,14 +68,20 @@ def report_status_for_branch(
             )
 
     @classmethod
-    def get_check_cls(cls, branch: str = None, identifier: Optional[str] = None) -> str:
+    def get_check_cls(
+        cls,
+        branch: str = None,
+        trigger_identifier: Optional[str] = None,
+        identifier: Optional[str] = None,
+    ) -> str:
         """
         Get name of the commit status for propose-downstream job for the given branch
         and identifier.
         """
         branch_str = f":{branch}" if branch else ""
+        trigger_str = f":{trigger_identifier}" if trigger_identifier else ""
```

Comment:
I haven't changed anything in the naming of the statuses for `propose-downstream`  (we haven't really discussed it). For this job, there is currently only one trigger (release); we were discussing in the past reacting to branch pushes, but have not planned this work. Do you think we can leave it as it is, and if we add a new trigger, we will change the naming only for that (so the `release` will be "default" similarly as `pull-request` for builds and tests)?

Reply:
but I realised this way, the parsing of the check-runs (in case of retriggering) needs to be differentiated, I will take a look into it